doc/api/vm.markdown

```diff
@@ -36,6 +36,12 @@ The options when creating a script are:
   code are controlled by the options to the script's methods.
 - `timeout`: a number of milliseconds to execute `code` before terminating
   execution. If execution is terminated, an [`Error`][] will be thrown.
+- `cachedData`: an optional `Buffer` with V8's code cache data for the supplied
+  source. When supplied `cachedDataRejected` value will be set to either
+  `true` or `false` depending on acceptance of the data by V8.
```

Comment:
CPU feature matching is a perennial problem in JVMs that support AOT compilation as well. Usually the solution for such AOT compilation is to produce the AOT code on the first invocation on a given machine and then persist it so that subsequent invocations can use the results of the first compilation. I am not sure the complexity is worth it in this case for Node, specially because the JVM class libraries tend to be much larger than the node core API, so AOT compilation has a bigger payoff on the JVMs.

This also doesn't help in scenarios where people run node inside containers where you don't expect repeated invocations of node.